website/src/supportedNetworks/NetworksTable.tsx

```diff
@@ -58,6 +58,51 @@ export function NetworksTable({ networks }: { networks: SupportedNetwork[] }) {
         </p>
       </Callout>
 
+      <div className="mb-6 overflow-clip rounded-8 border border-space-1500 bg-space-1800">
+        <div className="grid grid-cols-1 gap-px text-space-500 xs:grid-cols-2">
+          <div className="border-b border-r border-space-1500 p-4">
+            <Text.C10 className="mb-2 uppercase text-white">Subgraphs</Text.C10>
+            <div className="flex items-center gap-2">
+              <Check size={4} alt="Checkmark" />
+              <span className="text-14">{t('index.supportedNetworks.tableLegend.subgraphs.basic')}</span>
+            </div>
+            <div className="flex items-center gap-2">
+              <Checks size={4} alt="Checkmarks" />
+              <span className="text-14">{t('index.supportedNetworks.tableLegend.subgraphs.full')}</span>
+            </div>
+          </div>
+          <div className="border-b border-r border-space-1500 p-4 lg:border-r-0">
+            <Text.C10 className="mb-2 uppercase text-white">Substreams</Text.C10>
```

Comment:
I should have marked `Text` as `@deprecated` a while ago, my bad... but the new GDS way is the `text-c10` class.